src/strtree.h

```diff
@@ -5,13 +5,18 @@
 
 #include "geos.h"
 #include "pygeom.h"
-#include "vector.h"
 
+/* A resizable vector with addresses of geometries within tree geometries array */
+typedef struct {
+  size_t n, m;
+  GeometryObject*** a;
+} tree_geom_vec_t;
```

Comment:
The `geom_obj_vec_t` can be removed from vector.h, since it's no longer used? (unless you plan to use it elsewhere?)

Reply:
Can remove now, we can always add something back when we actually need it.